app/views/components/answers.scala.html

```diff
@@ -19,21 +19,21 @@
 
 @(answers: Seq[Answer], horizontal: Boolean, name: String)(implicit request: Request[_], messages: Messages)
 
-@answerText(answer: Answer) = @{ answer.txt }
+@answerText(answer: Answer) = @{ answer.text }
 
 <div class="govuk-radios@{if(horizontal) " govuk-radios--inline" else ""}">
   @for((ans, index) <- answers.zipWithIndex) {
 
     <div class="govuk-radios__item">
       <input class="govuk-radios__input"
             id="@name-item-@index"
-            name="questionPageUrl"
+            name="url"
             type="radio"
             value="@ans.dest"
             aria-describedby="@name-item-@index">
 
       <label class="govuk-label govuk-radios__label govuk-label--s" for="@name-item-@index">
```

Comment:
The Gov Uk document doesn't include the class govuk-label--s. Is this necessary?

Reply:
It looks as though the GovUk includes this when there is a hint text associated with the answer. I'll look further and will discuss with you